src/bin/tikv-server.rs

```diff
@@ -194,6 +218,23 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
     let wal_recovery_mode = util::config::parse_rocksdb_wal_recovery_mode(rmode).unwrap();
     opts.set_wal_recovery_mode(wal_recovery_mode);
 
+    let enable_statistics = get_boolean_value("",
+                                              "rocksdb.enable-statistics",
+                                              matches,
+                                              config,
+                                              Some(false),
+                                              |v| v.as_bool());
+    if enable_statistics {
+        opts.enable_statistics();
+    }
+    let stats_dump_period_sec = get_integer_value("",
```

Comment:
If enalbe_statistics is false, rocksdb also dump some statistics, if enalbe_statistics is true the statistics include more detail informations.